### PR TITLE
docs/4/update-README

### LINKED ISSUE #4: [DOC] Update README with npx example
# Incorrect Documentation

## Describe the error

There is a way to use this library by temporary install using `npx`.

## Expected information

An alternate example could include using `npx gimmie-colours` instead of the global install.

### PR DESCRIPTION
**Pull request by Subhangini**
# Pull Request

## Description:

 
Updated Usage section of README file. Added instruction to install the tool locally.

## Related Issue:

Closes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,10 @@ This page is currently deployed. [View the live website.](https://npmjs.com/pack
 
 ## Usage
 
-To install the tool, run `npm i -g gimme-colours`. Then run `gimme-colours` to see a colour-coded list of strings.
+**To install the tool**:-
+- globally:- run `npm i -g gimme-colours`. Then run `gimme-colours` to see a colour-coded list of strings.
+- locally:- run `npm i gimme-colours`. Then run `npx gimmie-colours` to see a colour-coded list of strings.
+>**NOTE**: global installation is preferred rather than local installation.
```
Comment (nhcarrigan) [inline README.md:16]:
```suggestion
**To install the tool**
- globally: run `npm i -g gimme-colours`. Then run `gimme-colours` to see a colour-coded list of strings.
- locally: run `npm i gimme-colours`. Then run `npx gimmie-colours` to see a colour-coded list of strings.
> **NOTE**: As this is a CLI tool, the global installation is preferred over the local installation. 
```

**Review by nhcarrigan — CHANGES_REQUESTED** [review]

**Review by nhcarrigan — APPROVED** [review]
Looks good to me.

**Comment by nhcarrigan** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

**Comment by Subhangini** [comment]
Thank you for assigning this issue to me and correcting me .

**Pull request merged**